docs/operator-dependencies-and-requirements.md

```diff
@@ -0,0 +1,40 @@
+# Operator Dependency and Requirement Resolution
+
+## Overview on Dependency Principles
+
+OLM manages the dependency resolution and upgrade lifecycle of running operators.
+
+An operator's dependencies are defined through the (Group, Version, Kind) of of an API. An operator cannot define a dependency on a specific version (e.g. `etcd-operator v0.9.0`) or instance (e.g. `etcd v3.2.1`) of an operator. This encourages operator authors to depend on the interface and not the implementation; thereby allowing operators to update on individual release cycles.
+
+OLM will resolve these dependencies by searching through existing `CatalogSources` for an operator that `owns` the API.
+
+OLM manages operator dependency resolution very similarily to OS package managers, such as apt/dkpg and yum/rpm. However, given that operators are always running, OLM attempts to ensure that it never deploys a combination of operators that do not work with each other.
+
+This means that OLM will never:
+
+* install a set of operators that require APIs that can't be provided
+* update an operator in a way that breaks another that depends upon it
+
+### OLM Dependency Resolution
+
+Dependency resolution begins when a `subscription` is reconciled by OLM.
```

Comment:
`Subscription` is an OLM object so it should be capitalized and put inside single quote just like `CatalogSource` and etc to keep the consistency across the doc.

Reply:
Addressed in the latest version of the PR.